UnityProject/Assets/Scripts/Core/Utils/MultiInterestString.cs

```diff
@@ -0,0 +1,6 @@
+using UnityEngine;
+
+public class MultiInterestString
```

Comment:
leftover?

Reply:
well is for when tongues have voices, in case you have multiple tongues xD

Reply:
yeah